main.go

```diff
@@ -57,6 +57,11 @@ func main() {
 		"Collect sources metrics",
 	).Default("false").BoolVar(&conf.CollectSources)
 
+	kingpin.Flag(
+		"collector.sources-with-ntpdata",
```

Comment:
Minor flag naming nit.
```suggestion
		"collector.sources.with-ntpdata",
```